docs/installing.md

```diff
@@ -1,6 +1,244 @@
+If you do not already have access to these keys, you can simply ignore them for now.
+
+```eval_rst
+.. note:
+    One secret key we have not covered thus far is WHEDON_SECRET.
+    This is because it is not one that you obtain from a provide,
+    but a secret key that you set yourself.
+    We recommend using something like a random SHA1 string.
+
+    It is important to remember this key,
+    as you will need it when deploying your Whedon-API application.
+```
+
+After pushing your application to Heroku, provisioning the appropriate add-ons,
+and confirming that your config variables are set correctly,
+you should make sure that your username is registered as an admin on the application.
+
+You can do this on a local Rails console, by logging in and setting the boolean field 'admin'
+on your user ID to True.
+If you'd prefer to do this on the Heroku deployment, make sure you've logged into the application.
+Then you can directly modify this attribute in the deployments Postgres database using SQL.
+For more information on accessing your application's Postgres database,
+see [the official docs](https://devcenter.heroku.com/articles/heroku-postgresql#pg-psql).
+
+## Making modifications to launch your own site
+
+Some times you may not want to launch an exact copy of JOSS, but a modified version.
+This can be especially useful if you're planning to spin up your own platform based on the
+Open Journals framework.
+[NeuroLibre](https://neurolibre.herokuapp.com) is one such example use-case.
+
+### Modifying your site configuration
+
+In this case, there are several important variables to be aware of and modify.
+Most of these are accessible in the `config` folder.
+
+First, there are three files which provide settings for your Rails application in different development contexts:
+
+1. `settings-test.yml`
+1. `settings-development.yml`
+1. `settings-production.yml`
+
+These each contain site-specific variables that should be modified if you are building off of the Open Journals framework.
+
+Next, you'll need to modify the `repository.yml` file.
+This file lists the GitHub repository where you expect papers to be published,
+as well as the editor team ID.
+For your GitHub organization, make sure you have created and populated a team called `editors`.
+Then, you can check its ID number as detailed in [this guide](https://fabian-kostadinov.github.io/2015/01/16/how-to-find-a-github-team-id).
+In `config` you should also modify the `orcid.yml` file to list your site as the production site.
+
+Optionally, you can edit `seeds.rb`, a file in the `db` folder.
+"DB" is short for "database," and this file _seeds_ the database with information about your editorial team.
+You can edit the file `seeds.rb` to remove any individuals who are not editors in your organization.
+This file will be updated later as you add information in the application, so this step is not strictly necessary.
```

Comment:
>This file will be updated later as you add information in the application

I don't think this happens. The `seeds.rb` file can be used to _initialize_ data in the database. It is not updated automatically by the application. The most common use for it is adding data and run it with `rails db:seed` or `rails db:setup` once to initialize/create the database. Then usually you don't need it again.

Reply:
Thanks ! I think I had manually edited the file initially and got a little confused.

I do wonder if it's worth being more explicit about this step, only as there were a couple times in development where I wanted to empty and re-create the database, and so I ended up using this sequence with the heroku CLI:

```
heroku pg:reset DATABASE_URL
heroku run rake db:schema:load
heroku run rake db:seed
heroku run rake searchkick:reindex:all
```

Do you think that's worth noting here, or somewhere else ?

Reply:
I think that's worth capturing.

Reply:
+1
Maybe favoring the `rails` command instead of `rake`:
```
heroku pg:reset DATABASE_URL
heroku run rails db:schema:load
heroku run rails db:seed
heroku run rails searchkick:reindex:all

```

Reply:
Should be fixed in a500273. Let me know what you think !